airbyte-config/init/src/main/resources/seed/destination_specs.yaml

```diff
@@ -3365,6 +3365,12 @@
             \ the sync. See the docs for details. Only relevant for COPY. Defaults\
             \ to true."
           default: true
+        use_super_redshift_type:
+          title: "Use SUPER Redshift Data-type for _airbyte_data column in _airbyte_raw table"
+          type: "boolean"
+          description: "This parameter is responsible for using SUPER column data-type during normalization. This\
+            \ option should increase performance during normalization, as well allows\
+            \ greater flexibility when creating source queries/views without normalization."
```

Comment:
Do we want to expose this option instead of just "upgrading" the behavior for all redshift users?
(even though forcing it may involve some migrations steps)

Reply:
about "Why do we implemented this as an option and not just forced to use SUPER type for all customers".

I have discussed this topic with Java Dev @alexandertsukanov who implemented redshift destination java part
https://github.com/airbytehq/airbyte/pull/9407

We formulate this such way:

1. It's a new feature that requires testing. Maybe it's good to keep both for some time?

2. IMPORTANT! If you store JSON in SUPER type (not in VARCHAR)
it requires from you to use completely different approach for SQL querying, for example:

for `VARCHAR`: `json_extract_array_element_text` function
for `SUPER`:  PartiQL extension over SQL

I can assume that some customers still would prefer to use old VARCHAR for JSON manipulation.

P.S.  if you really believe that we need to support only SUPER type
let us know we would think about migration steps.

Reply:
Hi, @ChristopheDuong!
I suggest two scenarios in this case:

1. Add the option for client to choose the type (Whether VARCHAR or SUPER).
2. Do not allow client to choose the option.

In first case, the are 2 strategies are possible:

- Remove tmp tables if they do not correspond chosen TYPE on UI and create new one which will be consistent.
- Disable the option to choose another option (e.g. disable SUPER toogler).

In the second case, we can proceed with the VARCHAR if tmp table was created with this datatype. Otherwise create SUPER datatype. (**Do not provide any options for client**). Please, let us know your opinion about that. Thanks.

CC: @antixar, @grubberr